tests/UiTestCase.php

```diff
@@ -60,32 +67,39 @@ protected function getSweetConfirmation($expectedText = null, $clickOk = true)
 
     protected function waitForElementPresent($target, $timeout = 60)
     {
-        for ($second = 0; ; $second++) {
-            if ($second >= $timeout)
+        for ($second = 0;; $second++) {
```

Comment:
hmm, not really a fan of this. Is there a different control structure we can use? How about a while(true)? That'd be more clear that it's meant to evaluate infinitely. Same for `waitForElementNotPresent`

Reply:
@LukeTowers Changed to `while` loops.